session/Session.proto

```diff
@@ -98,18 +97,28 @@ message Transaction {
     }
 
     message Iter {
-        message Req {
-            int32 id = 1;
-            bool all = 2;
-        }
         message Res {
             oneof res {
                 bool done = 1;
+                Batch.Res batch_res = 5;
+
                 Query.Iter.Res query_iter_res = 2;
                 GetAttributes.Iter.Res getAttributes_iter_res = 3;
                 Method.Iter.Res conceptMethod_iter_res = 4;
             }
         }
+        message Batch {
+            message Options {
+                int32 batch_size = 1;
+            }
```

Comment:
Are we adding this as a placeholder to add more options in the future?

Reply:
Kind of, it's because we need any options in two places (in `Query.Req` and `Iter.Batch.Req`) so I wanted to make sure that's properly captured now.

Reply:
👍